src/main/java/ch/njol/skript/conditions/CondIsVectorNormalized.java

```diff
@@ -0,0 +1,51 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.conditions;
+	
+	import org.bukkit.util.Vector;
+	import ch.njol.skript.conditions.base.PropertyCondition;
+	import ch.njol.skript.doc.Description;
+	import ch.njol.skript.doc.Examples;
+	import ch.njol.skript.doc.Name;
+	import ch.njol.skript.doc.RequiredPlugins;
+	import ch.njol.skript.doc.Since;
+
+@Name("Is Normalized")
+@Description("Checks whether a vector is normalized i.e. length of 1")
+@Examples("vector of player's location is normalized")
+@Since("INSERT VERSION")
+@RequiredPlugins("1.13 or newer")
```

Comment:
Looks like this was added in 1.13.2 from what I can see on the javadocs
```suggestion
@RequiredPlugins("1.13.2 or newer")
```